src/Registry/TypeRegistry.php

```diff
@@ -150,6 +150,13 @@ class TypeRegistry {
 	 */
 	protected $types;
 
+	/**
+	 * The keys that are prepared for introspection.
+	 *
+	 * @var array<string>|null
+	 */
+	protected static $introspection_keys = null;
```

Comment:
We're 7.4+ we can do cool stuff like this now 🚀🚀🚀

```suggestion
	protected static ?array $introspection_keys = null;
```